docs/index.html

```diff
@@ -0,0 +1,51 @@
+<!doctype html>
+<html>
+  <head>
+	<link rel="stylesheet" href="style.css">
+    <title>Arrowhead 5 | Java - Implementation</title>
+  </head>
+  <body>
+	<div id="title">
+		<div id="title-main">
+			ECLIPSE ARROWHEAD 5
+		</div>
+		<div id="title-sub">
+			Documentation of <strong>java-spring</strong> reference implementation
+		</div>
+	</div>
+	<div id="abstract">
+		<p>
+			The <strong>Eclipse Arrowhead Framework</strong> is addressing IoT based automation and digitalisation. The generic concept is based on the principles of Service Oriented Architectures, and aims at enabling all of its users to work in a common and unified approach – leading towards high levels of interoperability. The approach taken is that the information exchange of elements in the Internet of Things is abstracted to loosely coupled services. This is to enable IoT interoperability in-between almost any IoT elements.
+		</p>
+		<p>Learn more: <strong><a href="www.arrowhead.eu">www.arrowhead.eu</a></strong></p>
+	</div>
+	<div id="menu">
+		<div class="menu-item"><a href="./articles/getting-started">Getting started</a></div>
+		<div class="menu-item"><a href="./articles/client-libraries">Client libraries</a></div>
+		<div class="menu-item"><a href="./articles/tutorials">Tutorials</a></div>
+		<div class="menu-item"><a href="./articles/contribute">Contribute</a></div>
+	</div>
+	<div id="hero">
+		<div class="sys-group">
+			<div class="sys-group-title">Core System documents</div>
+			<div class="sys-group-list">
+				<a href="./serviceregistry/index"><div class="sys-group-list-item sr-color"><p>Service Registry</p></div></a>				
+				<a href="./orchestration/index"><div class="sys-group-list-item orch-color"><p>Service Orchestration</p></div></a>
+				<a href="./authorization/index"><div class="sys-group-list-item authorization-color"><p>Service Authorization</p></div></a>	
+				<a href="./authentication/index"><div class="sys-group-list-item authentication-color"><p>System Authentication</p></div></a>	
+			</div>
+		</div>
+		<div class="sys-group">
+			<div class="sys-group-title">Support System documents</div>
+			<div class="sys-group-list">
+				<div class="sys-group-list-item"><p>Event Handler</p></div>
+				<div class="sys-group-list-item"><p>Configuration</p></div>
+				<div class="sys-group-list-item"><p>Translation</p></div>
+				<div class="sys-group-list-item"><p>QoS Manager</p></div>
```

Comment:
This not the name of the support system. I don't remember the exact name but it was something like this: Default QoS Evaluator